docs/dev-remote.md

```diff
@@ -54,10 +54,10 @@ you cloned Zulip:
 This will start up the Zulip server on port 9991. You can then navigate to
 http://<REMOTE_IP>:9991 and you should see something like [(this screenshot of
 the Zulip development
-environment)](https://raw.githubusercontent.com/zulip/zulip/master/docs/images/zulip-dev.png).
+environment)](images/zulip-dev.png).
```

Comment:
due to an unfortunate sphinx issue, this resulted in a broken link.  I think the link here is unnecessary given the image is immediately after, so let's just remove the link.